<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrenchBowl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="prevBtn">
        <property name="text">
         <string>Prev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="toggleBtn">
        <property name="text">
         <string>Play/Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextBtn">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nowPlayingLbl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Artist - Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="posLbl">
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="seekBar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="durationLbl">
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="playlist">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
